Makefile

```diff
@@ -9,7 +10,10 @@ default: build
 install-crds:
 	kubectl apply -f deploy/crds/pulumi.com_stacks.yaml
 
-codegen: generate-k8s generate-crds
+codegen: install-controller-gen generate-k8s generate-crds
+
+install-controller-gen:
+	@echo "Installing controller-gen to GOPATH/bin"; pushd /tmp >& /dev/null && go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0 ; popd >& /dev/null
```

Comment:
I am using controller-gen 0.5.0 here because the old version and the latest 0.6.0 both mess with line wrapping on the descriptions for CRDs for some reason.